PurchasesTests/Networking/HTTPClientTests.swift

```diff
@@ -219,11 +218,10 @@ class HTTPClientTests: XCTestCase {
             )
         }
 
-        self.client.perform(request, authHeaders: [:]) { (status, data, error) in
-            correctResponse.value = (status.rawValue == errorCode) && (data != nil) && (error == nil)
-            if data != nil {
-                message.value = data?["message"] as? String
-            }
+        self.client.perform(request, authHeaders: [:]) { (status, result) in
+            correctResponse.value = (status.rawValue == errorCode) && (result.value != nil)
```

Comment:
[not for this PR] I feel like this variable would be more appropriately named `isResponseCorrect` or something that indicates it's a boolean, I kinda expected it to be an HTTPResponse instead

Reply:
Changed, and simplified the eventually calls too.